src/app/practice/page.tsx

```
@@ -0,0 +1,1205 @@
+                            marginBottom: '20px',
+                            padding: '15px',
+                            backgroundColor: '#f1f1f1',
+                            borderRadius: '6px',
+                            border: '1px solid #e0e0e0',
+                            fontSize: '15px',
+                            lineHeight: '1.6',
+                            maxHeight: '200px',
+                            overflowY: 'auto'
+                        }}
+                        // Dangerously set inner HTML as paragraph content can contain HTML tags
+                        dangerouslySetInnerHTML={{ __html: currentQuestion.question.paragraph }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)